test/utils/update_resources.go

```diff
@@ -48,9 +49,17 @@ func ScaleResourceWithRetries(scalesGetter scaleclient.ScalesGetter, namespace,
 		Size:            -1,
 		ResourceVersion: "",
 	}
+	exponentialBackoff := wait.Backoff{
```

Comment:
Maybe add a comment to tell what would be the backoff time at 1st, 6th and 12th attempts.

Reply:
Sure

Reply:
Currently the No of retries are 12 which is the Steps
Duration is the initial duration = 5s
Factor decides by how much times the duration of that current iteration increases here taken 2x
also here specified the Jitter it adds a small amount of balancing factor to this exponential growth to eliminate the race conditions

for 
iter 0 -> 5s
iter 1 -> 10s
iter 2 -> 20s
iter 3 -> 40s
...

Reply:
should we instead use the other factor or its looks good?
Currently
5 ( 1 + 2 + 2^2 + ... 2^(n-1)